etc/sources/file.yaml

```diff
@@ -3,7 +3,7 @@ default:
   fileType: json
   # The directory of the file relative to kuiper root or an absolute path.
   # Do not include the file name here. The file name should be defined in the stream data source
-  path: data
+  path: /opt/neuronex/data
```

Comment:
I am not sure if this shall be accepted. Making this change will affect a lot of eKuiper samples.

Reply:
@ngjaying Restored